integration/kubernetes/init.sh

```diff
@@ -65,10 +65,16 @@ export KUBECONFIG="$HOME/.kube/config"
 kubectl get nodes
 kubectl get pods
 
-# kube-flannel config file taken from k8s 1.12 documentation:
-flannel_config="https://raw.githubusercontent.com/coreos/flannel/bc79dd1505b0c8681ece4de4c0d86c5cd2643275/Documentation/kube-flannel.yml"
+arch=$("${SCRIPT_PATH}/../../.ci/kata-arch.sh")
+#Load arch-specific configure file
+if [ -f "${SCRIPT_PATH}/../../.ci/${arch}/kubernetes/init.sh" ]; then
+        source "${SCRIPT_PATH}/../../.ci/${arch}/kubernetes/init.sh"
+fi
+
+# default network plugin should be flannel, and its config file is taken from k8s 1.12 documentation:
+network_plugin_config=${network_plugin_config:-https://raw.githubusercontent.com/coreos/flannel/bc79dd1505b0c8681ece4de4c0d86c5cd2643275/Documentation/kube-flannel.yml}
```

Comment:
Not necessary for this PR as this code was already here, but I'm not very keen on hard-coding this commit value. Ideally, we should put it in https://github.com/kata-containers/tests/blob/master/versions.yaml and use `yq read` here to extract it.